README.md

```diff
@@ -21,6 +21,23 @@ application is sufficient to make the module available to your application.
 
 TODO: -->
 
+## Description
+
+ -Module provies wrapper to a credentials store (bedrock-credentials-mongodb for example).
+
+ -Angular module provides directive for credentials, and provides routes that map to templates that have a controller that will
```

Comment:
Looks like you might have an errant line break in here? Try to keep text to less than 75 characters wide in text files (it's easier for humans to read - our brains aren't good at following long lines - long lines also lead to eye fatigue).

This 75-character wide best practice is sometimes hard to follow with Markdown, so there are some cases where you can't do it (like in bulleted lists, IIRC).